gallery/src/components/NxGlobalSidebarFooter/NxGlobalSidebarFooterPage.tsx

```diff
@@ -89,7 +89,6 @@ export default function NxGlobalSidebarFooterPage() {
                   </a>
```

Comment:
While you're at it this link should be an `nx-text-link`

Reply:
> While you're at it this link should be an `nx-text-link`

I fixed it as part of RSC-576: https://github.com/sonatype/sonatype-react-shared-components/pull/397/commits/6e1820806f22030faad5cdc0f3bd725e404cbf99